README.md

```diff
@@ -43,6 +43,11 @@ info here](https://conferences.oreilly.com/software-architecture/sa-eu/public/sc
 
 ## Useful Information
 
+### Quick Start
+
+* [Quick-start guide](./docs/quick-start.md) explains how to get Styx up and running quickly
```

Comment:
Quick start guide has significant overlap with https://github.com/HotelsDotCom/styx/blob/master/docs/user-guide/basic-usage.md. Please consider merging the quick start guide into basic-usage.

Reply:
I think we should look into the overlap, but as a separate increment.